pkg/rbac/middleware.go

```diff
@@ -0,0 +1,97 @@
+package rbac
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/stolostron/search-v2-api/pkg/config"
+	authv1 "k8s.io/api/authentication/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+)
+
+var kClientset *kubernetes.Clientset
+
+func KubeClient() {
+	config, err := config.GetClientConfig()
+	if err != nil {
+		klog.Fatal(err.Error())
+	}
+	kClientset, err = kubernetes.NewForConfig(config)
+
+	if err != nil {
+		klog.Fatal(err.Error())
+	}
+
+}
```

Comment:
Could you move this block to `config/kubeClient.go` ?

Reply:
Yes, and we can remove the kube call from main too and directly call when we authenticate with the token review in the middleware.